components/Banner/docs/typical-use.md

```diff
@@ -4,7 +4,9 @@ By default, `MDCBannerView` is configured to display an image view, a text label
 
 ### Styling
 
-By default, `MDCBannerView` is configured to display items with black text and a white background. To customize the color and style of the text, image view and buttons displayed on `MDCBannerView`, directly set the relevant properties, such as `tintColor`, on `textView`, `imageView`, `leadingButton` and `trailingButton`.
+By default, `MDCBannerView` is configured to display items with black text and white background with a grey divider at the bottom of the view. To customize the color and style of the text, image view and buttons displayed on `MDCBannerView`, directly set the relevant properties, such as `tintColor`, on `textView`, `imageView`, `leadingButton` and `trailingButton`. `showsDivider` and `dividerColor` can be used to control the divider's visibility and color.
+
+`MDCBannerView` can handle its layout style in both an automatic way and manual ways. By default, `MDCBannerViewLayoutStyleAutomatic` is used and layout is set automatically based on how elements are configured on the `MDCBannerView`.
```

Comment:
Should we link to the enum and property that effect layout? Something to the effect:

_"To adjust a `MDCBannerView` layout use the `bannerViewLayoutStyle` property. This is set by default to `MDCBannerViewLayoutStyleAutomatic`"_

 I think just reading this, leads me to the thought, "of course it can adjust its layout I can just adjust my constraints/frame". That's just my opinion though, I could be looking at this through a skewed lens.

Reply:
It's definitely reasonable. All values are addressed in cd5eb9da57a656a6e270643f39a11972ad1ceba9 now. Do you feel if it addresses the clarity issue?